### ISSUE TITLE
bug report - rundeck user account 

### ISSUE THREAD

**Issue by SydOps** [puppet-community/puppet-rundeck#72]
I need rename default rundeck account from `rundeck` to other (following the company naming standard). 
```
rundeck::user: A1234
rundeck::group: A1234
```

After installed rundeck on a new system, two folders and subfolders are fine with user `A1234`: 
```
/etc/rundeck
/var/rundeck
```

But rundeck user is still created, and folder `/var/lib/rundeck` and subfolders are owned by rundeck. 
```
rundeck:x:502:502::/var/lib/rundeck:/bin/bash
```

My rundeck system is running on CentOS 6.6

**Comment by liamjbennett** [comment]
Based upon this:
https://github.com/rundeck/rundeck/blob/development/packaging/rundeck.spec#L140:L142

The problem is that the rundeck package creates this by default.

What needs to happen here is that we need to manage the rundeck user in `install.pp` and ensure that if the user != rundeck then we ensure absent for this default.

**Comment by SydOps** [comment]
Thanks for the confirmation @liamjbennett , that's the root cause. Seems in the same file, line 31~33 have the similar setting.

https://github.com/rundeck/rundeck/blob/development/packaging/rundeck.spec#L31:L33

**Comment by liamjbennett** [comment]
Fixed in 383d55d
